soc/nordic/nrf52/Kconfig

```diff
@@ -59,8 +59,7 @@ config SOC_DCDC_NRF52X_HV
 	  Enable nRF52 series System on Chip High Voltage DC/DC converter.
 
 config GPIO_AS_PINRESET
-	bool "[DEPRECATED] GPIO as pin reset (reset button)"
-	select DEPRECATED
+	bool
```

Comment:
`select DEPRECATED` should be kept, this option is still deprecated.

Reply:
When did we deprecate it? Should @nordicjm remove it altogether?